glide.yaml

```diff
@@ -1,5 +1,6 @@
-package: github.com/yard-turkey/vm-image-import
+package: kubevirt.io/containerized-data-importer
 import:
+- package: github.com/yard-turkey/vm-image-import
```

Comment:
? What is this used for? I don't see this repo in `github.com/yard-turkey`

Reply:
I have no idea, that was there, see the - on the left here. I am not using it, I just moved it around.

Reply:
@copejon any ideas?

Reply:
right .... that's some relic of CDI's early days.  We initially called CDI vm-image-import in yard-turkey before it was moved to the kubevirt org.  Grep shows 'yard-turkey' only appears in the .glide file, so we can remove that line

Reply:
Works for me, I will take it out.